src/AddonServiceProvider.php

```diff
@@ -32,7 +32,11 @@ public function boot(): void
         }
         
         if ($this->packageDirectoryExistsAndIsNotEmpty('resources/views')) {
-            $this->loadViewsFrom($this->path.'/resources/views', ':lc:vendor.:lc:package');
+            // Load published views
+            $this->loadViewsFrom(base_path('resources/views/vendor/:lc:vendor/:lc:package'), ':lc:package');
```

Comment:
```suggestion
            $this->loadViewsFrom(base_path('resources/views/vendor/:lc:vendor/:lc:package'), ':lc:vendor.:lc:package');
```

Agreed but let's use `vendor-name.package-name` as the namespace.